src/mte_tag.adoc

```diff
@@ -218,6 +218,11 @@ splitted stores.
   A memory access (load or store) to some virtual address `va` can not bypass
   the older store initiated by `settag/setinvtag rs1=va`.
 
+  This specification defines tag as the entitiy attached to virtual addresses.  
+  It is software's responsibility to ensure that the tags are appropriately set
+  on the virtual addresses even though some of them map to the same physical 
+  address(es).
```

Comment:
Can we re-word this

"""
  It is software's responsibility to ensure that the tags are appropriately set
  on the virtual addresses even though some of them map to the same physical 
  address(es)
"""

to something like

"""
In case of aliasing (multiple virtual addresses map to same physical address), it is software's
responsibility to ensure that the tags are set correctly for respective virtual address prior to
memory accesses via aliased virtual address.
"""

Reply:
Revised as suggested.